trillian/integration/hammer.go

```diff
@@ -1087,12 +1091,28 @@ func HammerCTLog(cfg HammerConfig) error {
 		glog.Info(s.String())
 	})
 
-	for count := uint64(1); count < cfg.Operations; count++ {
-		if err := s.retryOneOp(ctx); err != nil {
-			return err
+	if cfg.TestDuration == 0 {
```

Comment:
I think it might be nice to return a `please only set one of ...` error if `!(cfg.TestDuration == 0) ^ (cfg.Operations==0))`

Reply:
Done.

Reply:
It looks like the default value of the `--operations` flag will mean that if a user sets `--test_duration` they'll get a slightly confusing error message saying `cannot specify both test_duration and operations` even though they only specified one of the flags?

Reply:
Yes I just found that out. I can remove the default from operations if that won't confuse people already using it.

Reply:
A couple of other possibilities might be:
a) have hammer interpret cfg's type-default values to a useful default - i.e. hammer validates the passed in config and if neither ops nor duration is set then "override" ops to `^uint64(0)` to keep backwards compatibility
b) keep your current validation in hammer as is, but add the flags xor logic in main and have the opinion for what to do when no default is set expressed out there, but hammer requires you to have passed in an explicit cfg with a choice made.

Reply:
Or I could close this if the change is too disruptive and just set a large operation count.